docs/compatibility.md

```diff
@@ -21,6 +21,13 @@ You can find further information about released versions of ADF in the
 
 | ADF version | Content Services | Process Services |
 | --- | --- | --- |
+| [6.4.0] | v7.4.0 | v2.3.8 |
+| [6.4.0] | v7.3.0 | v2.3.5 |
+| [6.4.0] | v7.2.0 | v2.2.0 |
+| [6.3.0] | v23.1.0 | v2.4.2 |
+| [6.0.0] | v7.4.0 | v2.3.8 |
+| [5.1.0] | v7.3.0 | v2.3.5 |
+| [5.0.0] | v7.3.0-M2 | v2.3.4 |
```

Comment:
1. The ADF versions need to be linked to the corresponding entries in versionIndex.md, like in v4.2.0 and prior.
2. There are missing entries for the ADF versions 6.1.0, 6.2.0. 
3. Please confirm if there was an ADF release for v6.3.0. If so, then its changelog is missing in versionIndex.md
4. I'm not sure if there needs to be 3 separate entries for v6.4.0, although I might be wrong here

Reply:
1. VersionIndex for many versions do not exist.
2. @rbahirsheth, @garya123, kindly provide the understanding of the versions that need to be updated.
3. versionIndex for some versions is missing.
4. @rbahirsheth, @garya123, kindly provide the understanding of the versions that need to be updated.

Reply:
@swapnil-verma-gl 
Point no 1, is addressed.
Point no 2, I will discuss with Lovekesh & Garima
Point no 3, the versionIndex.md has been missed, As we discussed.
Point no 4, We have tested ADF 6.4.0 with different versions of ACS & APS.

Reply:
In this case while updating the compatibility matrix I think we should at least create `versionIndex.md` files for the versions added here, it shouldn't be too hard as each release should have release notes generated stating what's been changed, @MateuszHY what do you think?

Reply:
Good idea @MichalKinas

Reply:
updated the versionIndex for the missed ADF version.